local/http/http.go

```diff
@@ -111,13 +113,24 @@ func (s *Server) Start(errChan chan error) (int, error) {
 		return port, errors.WithStack(err)
 	}
 
+	var keyLogWriter io.Writer
+	if s.TlsKeyLogFile != "" {
+		w, err := os.OpenFile(s.TlsKeyLogFile, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0600)
+		if err != nil {
+			return port, errors.WithStack(err)
+		}
+
+		keyLogWriter = w
+	}
+
 	s.httpsserver = &http.Server{
 		Handler: proxyHandler,
 		TLSConfig: &tls.Config{
 			PreferServerCipherSuites: true,
 			CurvePreferences:         []tls.CurveID{tls.CurveP521, tls.CurveP384, tls.CurveP256},
 			Certificates:             []tls.Certificate{cert},
 			NextProtos:               []string{"h2", "http/1.1"},
+			KeyLogWriter:             keyLogWriter,
```

Comment:
Is there anything missing? I don't see where `KeyLogWriter` is used.

Reply:
@fabpot KeyLogWriter will be automatically handled by crypto/tls implementation in Go standard library. What we only have to set `io.Write` to this config field.

Reference: https://cs.opensource.google/go/go/+/refs/tags/go1.20:src/crypto/tls/common.go;drc=f9da938614c22c47239d3b78a06700b271fc621d;l=1346

Reply:
Oh, this is the tls configuration, sorry, I read too fast. All good here.